enatega-multivendor-app/src/screens/Register/Register.js

```diff
@@ -207,17 +210,27 @@ function Register(props) {
                       {country?.cca2}
                     </TextDefault>
                   </View>
-                  <TextInput
-                    placeholder={t('mobileNumber')}
+                  <View
                     style={[
                       styles(currentTheme).textField,
                       styles().phoneNumber,
                       phoneError && styles(currentTheme).errorInput
-                    ]}
-                    placeholderTextColor={currentTheme.fontSecondColor}
-                    value={phone}
-                    onChangeText={e => setPhone(e)}
-                  />
+                    ]}>
+                    <View
+                      style={{
+                        flexDirection: 'row',
+                        alignItems: 'center',
+                        paddingTop: 3
+                      }}>
```

Comment:
please move them to stylesheet, also use scale(3) for paddingTop

Reply:
done